backend/app/views/spree/admin/users/edit.html.erb

```diff
@@ -23,9 +23,22 @@
         </div>
       <% end %>
     <% end %>
+
   </div>
 </fieldset>
 
+<fieldset id='forgot-password'>
+  <legend><%= Spree.t(:forgot_password) %></legend>
+
+  <%= form_for [@admin, Spree.user_class.new], as: :spree_user, url: spree.admin_reset_password_path do |f| %>
+      <%= f.hidden_field :email, value: @user.email %>
+
+    <div class="filter-actions actions">
```

Comment:
We should just add a `align-center` class and should not make use of the `filter-actions actions` classes, as these will add even more borders to this already overwhelming page.